config/initializers/override_scoped_search_escaping.rb

```diff
@@ -5,6 +5,19 @@
 module ScopedSearch
   class QueryBuilder
     alias preprocess_parameters_old preprocess_parameters
+
+    # scoped_search gem does not preprocess values before passing to ext_method but sometimes it is necessary
+    # This method can be called from a scoped search ext_method to preprocess parameters
+    def self.preprocess_parameters(definition, field, operator, value)
+      inverted_operators = SQL_OPERATORS.invert.merge('ILIKE' => :like, 'NOT ILIKE' => :unlike)
```

Comment:
This is not dynamic, so you can export it to a constant outside the method and freeze it.